spec/rdkafka/admin_spec.rb

```diff
@@ -288,6 +288,9 @@
         expect(create_acl_report.rdkafka_response).to eq(0)
         expect(create_acl_report.rdkafka_response_string).to eq("")
 
+        # Since we create and immediately check, this is slow on loaded CIs, hence we wait
+        sleep(2)
```

Comment:
Can we move this to a method that will check if we are on a CI and sleep only then?

Reply:
No because it can also happen outside of CI once in a while. I would prefer stability over conditions.

Reply:
Sounds good. Can you adjust the comment cause it is misleading. Maybe move this to a method so we won't have to add sleeps every where and just call that method?

Reply:
ref https://github.com/karafka/karafka/issues/2041